quesma/quesma/config/config_v2.go

```diff
@@ -979,6 +883,14 @@ func (c *QuesmaNewConfiguration) getTargetsExtendedConfig(target any) ([]struct
 	}, 0)
 
 	if targets, ok := target.([]interface{}); ok {
+		for _, target := range targets {
+			if targetName, ok := target.(string); ok {
+				result = append(result, struct {
+					target     string
+					properties map[string]interface{}
+				}{target: targetName, properties: map[string]interface{}{}})
+			}
+		}
 		for _, target := range targets {
 			if targetMap, ok := target.(map[string]interface{}); ok {
 				for name, settings := range targetMap {
```

Comment:
I am worried that this messes up the order of targets in case some targets are `string`s and some are `map[string]interface{}` (not sure if it's possible). And the order of targets matters (for A/B testing for example).

You could easily solve this problem by moving those two ifs into a single loop, not two loops.

Reply:
ok, I was just thinking that this will be cleaner to have two passes, but it seems that your suggestion does not break tests.